gatsby-config.js

```diff
@@ -9,6 +9,20 @@ module.exports = {
     'MarkdownRemark.frontmatter.host': 'MarkdownRemark.frontmatter.location'
   },
   plugins: [
+    {
+      resolve: 'gatsby-plugin-manifest',
+      options: {
+        name: 'dsmJS',
+        short_name: 'dsmJS',
+        description: 'Des Moines JavaScript User Group',
+        lang: 'en',
+        start_url: '/',
+        background_color: '#EFDA4F',
+        theme_color: '#039998',
```

Comment:
this highlights that we still need to export brand colors from somewhere that is reusable. for now that probably means the components package, but eventually maybe a brand or design system package

Reply:
for awareness, i at least created [an issue](https://github.com/dsmjs/components/issues/574) to track getting this done at some point